README.md

```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
+| storage.endpoint                           | -                  | No|Optional. Specifies endpoint url for s3 compatible storages
+| storage.identity                           | -                  | Optional|Blob storage access key. Can be optional for filesystem, aws-s3, google-cloud-storage providers
+| storage.credential                         | -                  | Optional|Blob storage secret key. Can be optional for filesystem, aws-s3, google-cloud-storage providers
+| storage.bucket                             | -                  | No|Blob storage bucket
+| storage.overrides.*                        | -                  | No|Key-value pairs to override storage settings
+| storage.createBucket                       | false              | No|Indicates whether bucket should be created on start-up
+| storage.prefix                             | -                  | No|Base prefix for all stored resources. Must not contain path separators or any invalid chars
+| encryption.password                        | -                  | No|Password used for AES encryption
+| encryption.salt                            | -                  | No|Salt used for AES encryption
+| resources.maxSize                          | 1048576            | No|Max allowed size in bytes for a resource
+| resources.syncPeriod                       | 60000              | No|Period in milliseconds, how frequently check for resources to sync
+| resources.syncDelay                        | 120000             | No|Delay in milliseconds for a resource to be written back in object storage after last modification
+| resources.syncBatch                        | 4096               | No|How many resources to sync in one go
+| resources.cacheExpiration                  | 300000             | No|Expiration in milliseconds for synced resources in Redis
+| resources.compressionMinSize               | 256                | No|Compress a resource with gzip if its size in bytes more or equal to this value
+| redis.singleServerConfig.address           | -                  | No|Redis single server addresses, e.g. "redis://host:port"
+| redis.clusterServersConfig.nodeAddresses   | -                  | No|Json array with Redis cluster server addresses, e.g. ["redis://host1:port1","redis://host2:port2"]
+| invitations.ttlInSeconds                   | 259200             | No|Invitation time to live in seconds
+| access.admin.rules                         | -                  | No|Matches claims from identity providers with the rules to figure out whether a user is allowed to perform admin actions, like deleting any resource or approving a publication. Example: [{"source": "roles", "function": "EQUAL", "targets": ["admin"]}]. If roles contain "admin, the actions are allowed.
 
 #### Google IdP
 
 The setting `identityProviders.*.rolePath` should be set `fn:getGoogleWorkspaceGroups`.
```

Comment:
consider adding it in the description of identity provider parameters for a better usability